Configurations.md

```diff
@@ -964,6 +964,14 @@ fn add_one(x: i32) -> i32 {
 }
 ```
 
+## `format_generated_files`
+
+Format generated files. A file is considered as generated if the file starts with `// @generated`.
```

Comment:
Do you think it would be helpful/worthwhile here that those tools that are generating the files could also/alternatively insert `#![rustfmt::skip]` into the generated files, or would we rather steer the generated/skip scenarios to this approach?

Reply:
I don't think it's a good idea to encourage code generation tools to add `#![rustfmt::skip]`, as it entirely removes the ability to format the generated code.

Reply:
Yeah that makes sense. Some tools are doing that today because it's the only option, but the addition of this feature and the associated config options gives users a lot more power.